epub33/rs/index.html

```diff
@@ -280,280 +302,215 @@ <h2>EPUB Publications</h2>
 					as a distributed system.</p>
 			</div>
 		</section>
-		<section id="package-doc">
+		<section id="sec-package-doc">
 			<h2>Package Document Processing</h2>
 
-			<section id="sec-package-rs-conformance">
-				<h3>Package Document Conformance</h3>
+			<section id="pkg-doc-presentation">
+				<h4>Presentation Logic</h4>
 
-				<p>An <a>EPUB Reading System</a> is conformant with this section if meets all of the following
-					criteria:</p>
-
-				<ul class="conformance-list">
-					<li>
-						<p id="confreq-rendition-rs-package">It MUST honor all presentation logic expressed through the
-								<a href="https://www.w3.org/TR/epub-33/#sec-package-def">Package Document</a>
-							[[!EPUB-33]] (e.g., the reading order, fallback chains, page progression direction and fixed
-							layouts).</p>
-					</li>
-					<li>
-						<p id="confreq-rs-package-def">It MUST process the Package Document in conformance with all
-							Reading System conformance constraints expressed in <a href="#sec-package-def"></a>.</p>
-					</li>
-					<li>
-						<p id="confreq-rs-epub3-presentational-meta">It SHOULD process rendering metadata, as expressed
-							in <a href="#sec-package-metadata-rendering"></a>.</p>
-					</li>
-					<li>
-						<p id="confreq-rs-epub3-fxl-meta">It MUST process fixed layout metadata, as expressed in <a
-								href="#sec-fxl-props"></a>.</p>
-					</li>
-					<li>
-						<p id="confreq-rs-epub3-fxl-conflicts">It MUST ignore proprietary metadata properties that
-							pertain to layout expressions if they conflict behaviorally with the property semantics
-							defined in <a href="#sec-fxl-props"></a>.</p>
-					</li>
-					<li>
-						<p id="confreq-rendition-rs-manifest">It SHOULD NOT use non-<a>Publication Resources</a> in the
-							rendering of an EPUB Publication due to the inherent limitations and risks involved (e.g.,
-							lack of information about the resource and how to process it, security risks from
-							remotely-hosted sources, lack of fallbacks, etc.).</p>
-					</li>
-				</ul>
+				<p id="confreq-rendition-rs-package">It MUST honor all presentation logic expressed through the <a
+						href="https://www.w3.org/TR/epub-33/#sec-package-def">Package Document</a> [[!EPUB-33]] (e.g.,
```

Comment:
This is a new section; maybe it reads better if "an EPUB Reading System" is used instead of "It"...

Reply:
Oops, ya, missed that one.